README.md

```diff
@@ -152,7 +152,7 @@ At this point you will be able to run pgAdmin 4 from the command line in either
 server or desktop mode, and access it from a web browser using the URL shown in
 the terminal once pgAdmin has started up.
 
-Setup of an environment on Windows is somewhat more complicated unfortunately,
+Setting up of an environment on Windows is somewhat more complicated unfortunately,
```

Comment:
Hmm, tricky one. I think both are probably correct - though on reflection I might say "Setting up an environment on Windows"

Reply:
Yes. I just felt the "Setting up" is more suitable.